client/src/render.jsx

```diff
@@ -1,3 +1,5 @@
 import DescriptionService from './index.jsx';
+import React from 'react';
+import ReactDOM from 'react-dom';
 
-ReactDOM.render(<DescriptionService />, document.getElementById('description'));
\ No newline at end of file
+ReactDOM.hydrate(<DescriptionService initData={window.__initData__} initCurrent={'descriptionB'}/>, document.getElementById("description"));
```

Comment:
Interesting. Does hydrating still render? I need to look into this.

Reply:
it does!  it's used specifically to hydrate server side rendered html roots with data and play nicely with ReactDOMServer.  here: https://reactjs.org/docs/react-dom.html#hydrate